api/resource.rb

```diff
@@ -151,8 +155,8 @@ def validate
 
       check :collection_url_key, default: @name.pluralize.camelize(:lower)
 
-      check :create_verb, type: Symbol, default: :POST, allowed: %i[POST PUT]
-      check :read_verb, type: Symbol, default: :GET, allowed: %i[GET POST]
+      check :create_verb, type: Symbol, default: :POST, allowed: %i[POST PUT PATCH]
+      check :read_verb, type: Symbol, default: :GET, allowed: %i[GET POST PATCH]
```

Comment:
I will be _very_ concerned if we end up needing `PATCH` as a read verb.

Reply:
personally i dont think we should be validating these other than just having valid HTTP verbs

Reply:
Also :+1: on validating a shared set of all the common HTTP verbs (preferably broken out to a var or something)